packages/@okta/vuepress-site/docs/reference/import-hook/index.md

```diff
@@ -149,7 +150,13 @@ When you return an error object, it should contain an `errorSummary` sub-object:
 
 Returning an error object will cause Okta to record a failure event in the Okta System Log. The string you supplied in the `errorSummary` property of the `error` object will be recorded in the System Log event.
 
-## Sample JSON Payload of Request from Okta to Your External Service
+>**Note:** If a response to the Import Inline Hook request is not received from your external service within 3 seconds, a timeout occurs. In this scenario, the Okta import process continues and the user is created.
+
+#### Timeout behavior
```

Comment:
The timeout behavior is a pretty import thing to call-out - can we make this an H2 tag so it's shown prominently in the On This Page link explorer tree that appears on the right?

Reply:
Yes, but there was some pre-existing timeout and retry content already there. I've bundled it together and moved it toward the bottom. See attached screenshot for draft.
<img width="1259" alt="Screen Shot 2021-06-17 at 4 55 05 PM" src="https://user-images.githubusercontent.com/70648001/122471117-d9b7bb00-cf8c-11eb-9c6e-2fbf20ec8fd3.png">